README.md

```diff
@@ -1,158 +1,270 @@
+  <br />
+
+  <!-- APPENDICES/TALK2M DEV ID -->
+  <h3 id="obtaining-a-talk2m-developer-id"><span style="font-variant: small-caps;">Obtaining a Talk2M Developer ID</span></h3>
+  <p>For security reasons, a Talk2M Developer ID is required to access Talk2M services, and is required to use the Ewon Ignition Demo.</p>
+  <p> To obtain a Talk2M Developer ID and to learn more information, please visit the Talk2M Developer ID registration page at <a href="http://developer.ewon.biz/registration" class="uri">http://developer.ewon.biz/registration</a>.</p>
+  <br />
+
+  <!-- APPENDICES/ACCESSING EWON WEBPAGE -->
+  <h3 id="accessing-an-ewon-devices-webpage"><span style="font-variant: small-caps;">Accessing an Ewon Device’s Webpage</span></h3>
+  <p>To access your Ewon device’s webpage, you’ll need to connect your PC to your Ewon’s LAN. If you know the IP address of your Ewon device, you can connect to the Ewon device’s webpage by entering its IP address in your web browser. If you do not know its IP address, or you are unsure, launch eBuddy and locate your Ewon device in the list. Then, right click the Ewon device and select <code>Open in Browser</code>.</p>
+  <br />
+
+  <!-- APPENDICES/INSTALLING IGNITION -->
+  <h3 id="installing-and-configuring-ignition"><span style="font-variant: small-caps;">Installing and Configuring Ignition</span></h3>
+  <p>To download Ignition, please visit the Induction Automation downloads webpage in your web browser at <a href="https://inductiveautomation.com/downloads" class="uri">https://inductiveautomation.com/downloads</a>.</p>
+  <p>For information on how to install Ignition and access Ignition’s web page, please visit the Ignition quick start guide at <a href="https://docs.inductiveautomation.com/display/DOC80/Quick+Start+Guide" class="uri">https://docs.inductiveautomation.com/display/DOC80/Quick+Start+Guide</a>.</p>
+  <p><strong>NOTE:</strong> The Ignition Quick Start Guide contains various sections, some of which are not related to the Ewon Ignition Demo. For the Ewon Ignition Demo, it is only necessary to follow the instructions and information found in section 1, “Download and Install Ignition” and section 2, “Launch the Gateway Webpage.” During Ignition installation, you may choose a “Typical” installation.</p>
+  <br />
+
+  <!-- APPENDICES/RESTARTING IGNITION -->
+  <h3 id="restarting-ignition"><span style="font-variant: small-caps;">Restarting Ignition</span></h3>
+  <p>Ignition can be restarted using two methods, listed below.</p>
+  <p><strong>Method 1 (Easiest):</strong> Restart your PC.</p>
+  <p><strong>Method 2 (Advanced)</strong>: Browse to your Ignition gateway’s installation directory, and run the stop-ignition.bat script to stop the Ignition gateway process. Once the stop-ignition.bat script finishes running and closes, run the start-ignition.bat script to start the Ignition gateway process again.</p>
```

Comment:
This is windows specific, maybe break out the instructions for each OS.  I think linux is the same but they are just .sh files

Reply:
updated to include linux, and more friendly direction